### ISSUE TITLE
1.21 version missing on Modrinth

### ISSUE THREAD

**Issue by HaXrDEV** [Khajiitos/ServerCountryFlags#16]

**Comment by LightWilliamz** [comment]
Is dev abandoned this project?

**Comment by HaXrDEV** [comment]
> Is dev abandoned this project?

I don't think so. The mod is available for 1.21 on CurseForge, just not Modrinth.

**Comment by Wudji** [comment]
https://github.com/Khajiitos/WorldPlayTime/issues/9

**Comment by Khajiitos** [comment]
It's been a while, 1.21 versions have been available on Modrinth for some time now
